BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -0,0 +1,11 @@
+# PV Database records
+
+record(bo, "$(P)$(R)RFFERst-Cmd") {
+    field(DESC, "Reset RFFE")
+    field(DTYP, "stream")
+    field(SCAN, "Passive")
+    field(VAL, "0")
+    field(ZNAM, "reset")
+    field(ONAM, "reset")
```

Comment:
In the end, I removed `ZNAM`, because it is the state used by the new `RFFERst` record to decide whether a `SYSTem:RESet` will be sent. This way, the PV has no label when its not currently resetting, and it has the `reset` label during the second (`HIGH = 1`) that follows the `RFFERst-SP` (future `-Cmd`) processing.